src/index.ts

```diff
@@ -40,3 +40,4 @@ export * from "./apollo/utils/X25519KeyPair";
 export * from "./pollux/models/AnonCredsVerifiableCredential";
 export * from "./pollux/models/JWTVerifiableCredential";
 export { KeyProperties } from "./domain/models/KeyProperties";
+export * from './pollux/AnoncredsLoader';
```

Comment:
this seems odd, why are we exporting AnoncredsLoader, which should be internal to Pollux?

Reply:
solved